GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1525,7 +1525,7 @@ private void RebaseToolStripMenuItemClick(object sender, EventArgs e)
         {
             var revisions = RevisionGrid.GetSelectedRevisions();
 
-            if (revisions.Count == 0)
+            if (revisions.Count == 0 || revisions[0].IsArtificial)
```

Comment:
Note for other reviewers: This check is not in GitUICommands, as the string is passed, not the ObjectId so each caller must check.